packages/@lightningjs/ui-components/src/components/InlineContent/InlineContent.stories.js

```diff
@@ -231,3 +231,37 @@ WithTruncation.args = {
   maxLines: 2,
   maxLinesSuffix: '...'
 };
+
+export const WithStrikeThrough = args =>
+  class WithParsing extends lng.Component {
+    static _template() {
+      return {
+        InlineContent: {
+          type: InlineContentComponent,
+          x: 550,
+          w: 400,
+          contentWrap: args.contentWrap,
+          contentProperties: args.contentProperties,
+          justify: args.justify,
+          customStyleMappings: {
+            italic: {
+              fontStyle: 'italic',
+              textColor: getHexColor('FF6194')
+            }
+          },
```

Comment:
This can be removed since this style isn't being used in this story.

Reply:
Removed